### PR TITLE
Adds syntax highlighting

### LINKED ISSUE #487: Add style for code blocks [$20]
We should copy the solarized-like style we had on the old docs. It was rad

<bountysource-plugin>
---
There is a **[$20 open bounty](https://www.bountysource.com/issues/10949023-add-style-for-code-blocks?utm_campaign=plugin&utm_content=tracker%2F10548672&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F10548672&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by emersion**
Fixes #487.

It's using [highlight.js](https://highlightjs.org). Included languages: Vala, CMake, Bash, Makefile.

screenshot from 2015-04-15 21-11-44

### CODE REVIEW

**Comment by emersion** [comment]
Light theme:

screenshot from 2015-04-15 21-15-30

**Comment by emersion** [comment]
screenshot from 2015-04-15 21-52-38

**Comment by danrabbit** [comment]
That looks ace dude. Visual design approve :D

File: styles/main.css [hunk 1]
```diff
@@ -164,6 +164,31 @@ pre > code {
     white-space: pre;
 }
 
+/* Line numbers */
+pre.highlighted {
+    border: none;
+    padding: 0;
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```
Comment (danrabbit) [inline styles/main.css]: Break line after the "," please

Comment (lewisgoddard) [inline styles/main.css]: Break and indent or just break?

**Pull request merged**